### PR TITLE
Empty State UI of Overview Page

### PR DESCRIPTION
**Pull request by MVarshini**
PBENCH-990

Empty State UI of Overview Page

Update the no result view of overview page

### CODE REVIEW

File: dashboard/src/modules/components/OverviewComponent/index.jsx [hunk 1]
```diff
@@ -84,8 +87,14 @@ const OverviewComponent = () => {
                 New and Unmanaged
               </AccordionToggle>
               <AccordionContent isHidden={!expanded.has("newRuns")}>
-                <NewRunsHeading />
-                <NewRunsComponent />
+                {newRuns.length > 0 ? (
+                  <>
+                    <NewRunsHeading />
+                    <NewRunsComponent />
+                  </>
```
Comment (webbnh) [inline dashboard/src/modules/components/OverviewComponent/index.jsx:94]: We didn't seem to need the empty tag at (new) line 91 before...so why do we need it now?  (I.e., what happens if we omit it?)

File: dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx [hunk 2]
```diff
@@ -148,7 +148,7 @@ const NewRunsComponent = () => {
               const isItemFavorited = !!item?.metadata?.[USER_FAVORITE];
               const isItemSeen = !!item?.metadata?.[DASHBOARD_SEEN];
               return (
-                <Tbody key={rowIndex}>
+                <Tbody key={rowIndex} isExpanded={isRunExpanded(item)}>
```
Comment (webbnh) [inline dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx:151]: So, new runs are "expandable", but saved runs and expiring runs are not?

**Review by webbnh — DISMISSED** [review]
Looks OK to me; I just have a couple of questions.

**Review by portante — DISMISSED** [review]

**Review by portante — APPROVED** [review]

**Review by dbutenhof — APPROVED** [review]

File: dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx (hunk 2 above)
Comment (webbnh) [reply dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx:151]: > Can you help me understand how you arrived at that conclusion?

I compared the line here to the [analogous line](https://github.com/distributed-system-analysis/pbench/blob/18465a897ce573cd648b9788b0191da2bc9b791a/dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx#L115) in the saved-run code:  the `isExpanded` prop is being added here but not there, so I figured I would inquire about the apparent lack of symmetry.

File: dashboard/src/modules/components/OverviewComponent/index.jsx (hunk 1 above)
Comment (webbnh) [reply dashboard/src/modules/components/OverviewComponent/index.jsx:94]: Thanks, Varshini!

**Pull request merged**